### ISSUE TITLE
Graphical issues with using `squiggly`

### ISSUE THREAD

**Issue by varkor** [jonschz/tikz-nfold#4]
```tex
\begin{tikzcd}
	\bullet & \bullet
	\arrow[Rightarrow, squiggly, from=1-1, to=1-2]
\end{tikzcd}
```
Without `nfold`:
<img width="114" alt="image" src="https://user-images.githubusercontent.com/3943692/230623532-5118bcd4-b249-411f-823c-d7501e89a035.png">
With `nfold`:
<img width="109" alt="image" src="https://user-images.githubusercontent.com/3943692/230623578-6ac0646f-388b-4c95-bbac-668bbd8069cc.png">

**Comment by varkor** [comment]
It's possible this issue is similar to the one I encountered in https://github.com/varkor/quiver/issues/94.

**Comment by jonschz** [comment]
Thanks again for testing and reporting. After some debugging I found that there are multiple bugs involved here, some of which boil down to the (frankly quite abusive) use of the `decorations` library. Some remedies are possible, but I am hoping that #7 will fix many of these problems on its own.

**Comment by jonschz** [comment]
This is now fixed on the development branch for #7 :
![grafik](https://user-images.githubusercontent.com/17198703/231751593-b0bfb54a-1d7f-4d19-90b6-282bc36cbbd1.png)
